lib/formulator.ex

```diff
@@ -2,31 +2,80 @@ defmodule Formulator do
   use Phoenix.HTML
   alias Formulator.HtmlError
 
-  def input(form, field, options \\ []) do
+  @doc  """
+  Returns an html input with an associated label.
+
+  Passing `label: false` into the options will return the input without a label.
+  In that case it will add an `aria-label` attribute to the input to improve
+  accessibility.
+
+  ## Examples
+
+  Basic input:
+      <%= input form, :email %>
+      #=> <label for="user_email">Email</label>
+      #=> <input id="user_name" name="user[email]" type="text" value="">
+
+  Without a label:
+      <%= input form, :email, label: false %>
+      #=> <input id="user_name" name="user[email]" aria-label="email" type="text" value="">
+  """
+
+  @spec input(Phoenix.HTML.Form.t, atom, []) :: binary
+  def input(form, field, options \\ [])
```

Comment:
As discussed, you probably don't need this since you're only defining input once. But if you think it's nice to have for the future I'm only at like a 5.5 in wanting this to go